scripts/webpack.config.js

```diff
@@ -28,6 +29,11 @@ module.exports = {
       logLevel: 'error',
       openAnalyzer: false,
     }),
+    new DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(isProd ? 'production' : 'development'),
```

Comment:
this should probably be `JSON.stringify(process.env.NODE_ENV || 'development')` instead